src/index.tsx

```diff
@@ -25,21 +26,21 @@ const queryClient = new QueryClient({})
 
 const App = (): JSX.Element => {
   return (
-    <SafeAreaProvider>
```

Comment:
Why did you remove SafeAreaProvider?

Reply:
That provider isn't needed as `NativeBaseProvider` already wraps its children with `SafeAreaProvider`.

Maybe it would be good to leave a comment in a render tree informing that `SafeArea` features are available due to `NativeBase` integration?

Reply:
It's good to know that! :D 

Yes, it's good idea to add some informations about SafeAreaProvider because it can be confusing for new developers